lib/pos_blockchain.go

```diff
@@ -506,8 +509,19 @@ func (bc *Blockchain) getLineageFromCommittedTip(desoBlock *MsgDeSoBlock) ([]*Bl
 		if currentBlock.IsCommitted() {
 			return nil, RuleErrorDoesNotExtendCommittedTip
 		}
+		if currentBlock.IsValidateFailed() {
+			return nil, RuleErrorAncestorBlockValidationFailed
+		}
+		if currentBlock.Header.ProposedInView >= prevView {
+			return nil, RuleErrorAncestorHasGreaterOrEqualView
```

Comment:
Maybe this is tiny more descriptive of what's happening?
```suggestion
			return nil, RuleErrorParentBlockHasViewGreaterOrEqualToChildBlock
```